docker/example.env

```diff
@@ -12,6 +12,7 @@ DB_DATA_LOCATION=./postgres
 IMMICH_VERSION=release
 
 # Connection secret for postgres. You should change it to a random password
+# Please use only the characters `A-Za-z0-9`, without special characters or spaces
```

Comment:
Shouldn't we rather recommend just quoting the password?

Reply:
We could. If we did, we should add single quotes to the default docker-compose.yml and specify that single quotes are an unallowed value.

This seemed easier to me, because I helped someone on Discord earlier this week who had the password set to (snippet of PW): `\æ¨Þèùdïîû`

I don't really know if everything downstream will work with such a password tbh

Reply:
OMG lmao. Ok I'm definitely happy with this then lmao
It's not like the DB should be exposed anyways